Underanalyzer/VMData.cs

```diff
@@ -761,7 +793,7 @@ public interface IGMVariable
 
     /// <summary>
     /// The ID of the variable in the game's data file.
-    /// This can sometimes be -6, representing the instance type of a built-in variable.
+    /// This can sometimes be -6, representing the instance type, of a built-in variable.
```

Comment:
Not sure why this comma was added

Reply:
to make built-in IDE grammar tool shut up lmao